geode-docs/tools_modules/geode_apis_compatible_with_redis.html.md.erb

```diff
@@ -68,6 +68,7 @@ The <%=vars.product_name%> APIs compatible with Redis currently supports the fol
 
 **Note**: These commands are supported for Redis 5.
 **Note**: SLOWLOG is implemented as a NoOp for compatibility with some monitoring tools.
+**Note**: Redis accepts Longs as the parameters for the HSCAN command. The Geode APIs compatible with Redis only accept Integer values. Parameters out of the range of the 32-bit integer type will throw an exception.
```

Comment:
Forgot to mention, @karensmolermiller previously made a comment in [this PR](https://github.com/apache/geode/pull/6149) regarding the command notes:
```
If there will be more notes about specific commands, I'd like to see the notes placed next to the command within the lists. Maybe add the note in parenthesis? I believe that documentation readers are more likely to see the note if it is with the command, as I believe that readers will be likely to scan this page for the commands that they use. I don't think they'll be starting at the top to do a detailed reading.
```

Reply:
I definitely agree with that but the current format doesn't make that very easy. [this PR](https://github.com/apache/geode/pull/6239/) changes the format of this list, which will allow an inline note. I will make this change when I rebase that command on top of this (once it is merged)